docs/private-dns/connect-devices/mobile-and-desktop/linux.md

```diff
@@ -104,19 +104,19 @@ If you see a notification that you are not connected to AdGuard DNS, most likely
 
 ## Use EDNS (Extended DNS)
 
-EDNS extends the DNS protocol, enabling larger UDP packets to carry additional data. In AdGuard DNS, it allows passing the DeviceID in plain DNS using an extra parameter.
+EDNS extends the DNS protocol, enabling larger UDP packets to carry additional data. In AdGuard DNS, it allows passing DeviceID in plain DNS using an extra parameter.
 
 DeviceID, an eight-digit hexadecimal identifier (e.g., `1a2b3c4d`), helps link DNS requests to specific devices. For encrypted DNS, this ID is part of the domain (e.g., `1a2b3c4d.d.adguard-dns.com`). For unencrypted DNS, EDNS is required to transfer this identifier.
 
-AdGuard DNS uses EDNS to retrieve the DeviceID by looking for option number 65074. If such an option exists, it will read the DeviceID from there. For this, you can use the command `dig` on the terminal:
+AdGuard DNS uses EDNS to retrieve DeviceID by looking for option number `65074`. If such an option exists, it will read DeviceID from there. For this, you can use the `dig` command on the terminal:
```

Comment:
```suggestion
AdGuard DNS uses EDNS to retrieve DeviceID by looking for option number `65074`. If such an option exists, it will read DeviceID from there. For this, you can use the `dig` command in the terminal:
```